tests/pet.test.js

```diff
@@ -109,4 +113,15 @@ describe('checks if pet is alive',() => {
     pet.hunger = 9;
     expect(pet.isAlive).toBeTruthy();
   });
+});
+
+describe('child constructor', () => {
+  it('returns an object', () => {
+    expect(new Pet('Amelia')).toBeInstanceOf(Object);
+  });
+  it('returns child array', () => {
```

Comment:
I would suggest to fix the code as stated above, and then check if the length of the array changes if you call the adoptChild twice. With the current code, what would be the result?
And what if you change it to use the .push method?